core/vm/logger.go

```diff
@@ -314,9 +305,68 @@ func (l *StructLogger) CaptureEnd(output []byte, gasUsed uint64, t time.Duration
 }
 
 func (l *StructLogger) CaptureEnter(typ OpCode, from common.Address, to common.Address, input []byte, gas uint64, value *big.Int) {
+	//the last logged op should be CALL/CREATE
```

Comment:
should it be `CALL/CALLCODE/CREATE/`?

Reply:
there are five types of CALLxx. So either we list them all, or just put `CALL` here. I think a normal reader will understand all CALL(staticcall delegatecall call callcode..) should share same logic. They have no reason not share some procedure.

Reply:
Yes, I simply missed CALLCODE ...
I would double check here and some related code